tests/test_nightReporter.py

```diff
@@ -30,27 +30,23 @@
 
 from lsst.summit.extras.nightReport import NightReporter, loadReport, saveReport  # noqa: E402
 import lsst.summit.utils.butlerUtils as bu  # noqa: E402, N813
-import lsst.daf.butler as dafButler  # noqa: E402, N813
 
 
 class NightReporterTestCase(lsst.utils.tests.TestCase):
 
     @classmethod
     def setUpClass(cls):
         try:
-            cls.butler = dafButler.Butler('LATISS', instrument='LATISS', collections=['LATISS/raw/all'])
+            cls.butler = bu.makeDefaultLatissButler()
```

Comment:
The short module name might look like a typo (`bu` instead of `butler), and doesn't agree with https://developer.lsst.io/python/style.html#import-statements-should-follow-a-standard-pattern  I understand this is a sitcom package, and so that potentially doesn't apply, but even `butlerUtils` seems to be better.

Reply:
Yup, totally agree 👍